server/player.lua

```diff
@@ -316,6 +316,8 @@ function CreatePlayer(playerData, Offline)
         if meta == 'hunger' or meta == 'thirst' then
             val = val > 100 and 100 or val
         end
+        TriggerClientEvent('qbx_core:client:onSetMetaData', self.PlayerData.source, meta, self.PlayerData.metadata[meta], val)
+        TriggerClientEvent('qbx_core:server:onSetMetaData', meta, self.PlayerData.metadata[meta], val)
```

Comment:
Move these to be the last thing in the function. Keep in mind you'll need to save self.PlayerData.metadata[meta] in a variable to save it before overwriting. That way these events will only trigger after the update playerdata event.